package.json

```diff
@@ -63,7 +63,7 @@
     "moment-strftime": "~0.5.0",
     "moment-timezone": "~0.5.21",
     "numeral": "~2.0.6",
-    "patternfly": "~3.59.1",
+    "patternfly": "~3.59.4",
```

Comment:
Can you revert it back to the original version? I think this upgrade did not help anything.